command/connect/proxy/proxy.go

```diff
@@ -405,14 +405,16 @@ func (c *cmd) Help() string {
 	return c.help
 }
 
-const synopsis = "Runs a Consul Connect proxy"
+const synopsis = "Runs a non-production, built-in service mesh sidecar proxy"
```

Comment:
The docs explicitly state that this isn't meant to be a production sidecar proxy. So I've also added that to the help text.